AlpsSDK/ProximityHandler.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  ProximityHandler.swift
+//  AlpsSDK
+//
+//  Created by Wen on 26.10.17.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+import Alps
+
+class ProximityHandler {
+    
+    var contextManager: ContextManager!
+    static var immediateTrigger: [String: ProximityEvent] = [:]
```

Comment:
as discussed, it's better to have a list of beacons and filter for those that match our proxmity requirement.

It will let us remove all structures below and end up with list and filter params.